sm_modified_cnn/README.md

```diff
@@ -25,6 +29,7 @@ We use `trec_eval` for evaluation:
 ```bash
 cd eval
 tar -xvf trec_eval.9.0.tar.gz
+cd trec_eval.9.0 
 make
 cd ..
```

Comment:
Maybe you want to change the line 34 to cd ../..

Reply:
Oops, missed it! Thanks!